python/cuml/cluster/kmeans.pyx

```diff
@@ -450,6 +480,7 @@ class KMeans(Base):
                 <size_t> self.n_cols,          # n_features (cols)
                 <int> 0,                       # distance metric as squared L2: @todo - support other metrics # noqa: E501
                 <int*> labels_ptr,             # pred_labels
+                <double*> address(inertia),    # inertia value
```

Comment:
Why use this instead of just `&inertia`? The `&` is the standard we've been using and as far my knowledge `address(*)` has no advantage over it